Chat/Parcer.h

```diff
@@ -15,8 +15,8 @@ enum ConstantSizes
     PREPARE_MESSAGE_SIZE = 5,   
```

Comment:
they are constantly changing and im not sure how to name such constant if i made one (as two of them represent constant header size of message and to constant actual size of message)
enums declaration don't use any memory (if i remember correctly), and in this way it is better for readability